src/turbopuffer.ts

```diff
@@ -423,16 +423,16 @@ function toColumnar(vectors: Vector[]): ColumnarVectors {
 }
 
 function fromColumnar(cv: ColumnarVectors): Vector[] {
-  const res = new Array<Vector>(cv.ids.length);
+  const res = new Array<Vector>(cv.ids?.length);
```

Comment:
change

Reply:
assuming passing undefined to an array constructor just defaults to 0-length array?